src/Faithlife.Utility/InvariantConvert.cs

```diff
@@ -113,6 +113,30 @@ public static string ToInvariantString(this double value)
 		/// <exception cref="FormatException">Failed to parse using invariant culture.</exception>
 		public static TimeSpan ParseTimeSpan(string text) => ThrowFormatExceptionIfNull(TryParseTimeSpan(text), text);
 
+		/// <summary>
+		/// Converts the value to a string using the invariant culture.
+		/// </summary>
+		public static string ToInvariantString(this object value)
```

Comment:
I'm not sure we want this extension method on every object.

```suggestion
		public static string ToInvariantString(object value)
```